src/controllers/tasks/active.psc.details.controller.ts

```diff
@@ -14,13 +14,18 @@ import {
   toUpperCase
 } from "../../utils/format";
 import { toReadableFormat } from "../../utils/date";
+import { logger } from "../../utils/logger";
 
 export const get = async (req: Request, res: Response, next: NextFunction) => {
   try {
     const transactionId = urlUtils.getTransactionIdFromRequestParams(req);
     const submissionId = urlUtils.getSubmissionIdFromRequestParams(req);
     const pscs: PersonOfSignificantControl[] = await getPscs(req.session as Session, transactionId, submissionId);
-
+    if (pscs.length < 1) {
```

Comment:
pscs might be undefined as well, i would include that in the if check like
if (!pscs || pscs.length < 1)